.github/workflows/aa_cc_kbc.yml

```diff
@@ -1,6 +1,8 @@
 name: CC kbc build CI
 on:
   push:
+    branches:
+      - "main"
```

Comment:
I found others are `'main'`. Does this matter? s.t. `'` and `"`

Reply:
I don't think it makes a difference, but I updated it anyway for consistency.